drivers/FileHandle.h

```diff
@@ -91,7 +114,8 @@ class FileHandle {
      *    new file position on success,
      *    -1 on failure or unsupported
      */
-    virtual off_t lseek(off_t offset, int whence) = 0;
+    MBED_DEPRECATED_SINCE("mbed-os-5.4", "Replaced by FileLike::seek")
+    virtual off_t lseek(off_t offset, int whence) { return seek(offset, whence); }
```

Comment:
The "l" in lseek stands for "long", to differentiate early versions of the function from its 16-bit counterpart. I'm not sure this is useful anymore, especially in c++.